### ISSUE TITLE
Idea: Ability to randomise teams.

### ISSUE THREAD

**Issue by ChuckleSauce** [slicedlime/Sacrifice#18]
Hey I know your a busy man and completely understand if this is ignored....

A way to randomise the team members at the start, would be very helpful. I think the ability to set team sizes would be needed, odd numbers will be difficult, but maybe it could leave the odd one out (chosen at random) and then can manually join a team after the randomisation before starting the game?

I use Chosen as a way to introduce new members to my SMP and have even organised 2 SMPs together, and doing this in game instead of using a web tool would be much better and simpler, if you have 10 or more people playing its like hearding cats.

And thanks for such an awesome game! Everyone that has played with me loves it, and I get a lot of requests to play more often.

**Comment by BraddlesGaming** [comment]
Excellent idea! A randomiser would come in very handy!

**Comment by slicedlime** [comment]
Yes - this is a thing that has been on my todo list for quite a while. I made a randomization method for Mob Dash, but it never made its way back to Chosen. Hopefully I can make something reusable.
